contents/docs/cdp/troubleshooting.md

```diff
@@ -70,4 +76,10 @@ We recommend sending a `uuid` value with every captured event. Events with the s
 
 This is important because failures and retries happen, so your application or our library might send the same event multiple times. If you don't send UUIDs for every event, we aren't able to know if it's the same event, and hence we are not able to de-duplicate it for you.
 
-Some of our SDKs, such as JavaScript Web, do this automatically, other SDKs allow you to pass UUIDs. If you're using an SDK which doesn't currently support UUIDs for every event, please consider creating a PR or filing an issue in GitHub.
\ No newline at end of file
+Some of our SDKs, such as JavaScript Web, do this automatically, other SDKs allow you to pass UUIDs. If you're using an SDK which doesn't currently support UUIDs for every event, please consider creating a PR or filing an issue in GitHub.
+
+## Solved community questions
```

Comment:
<img width="690" height="160" alt="Screenshot 2025-12-29 at 11 23 25 AM" src="https://github.com/user-attachments/assets/dad570d1-6064-4f3c-a666-847b4fe52da4" />
lol top question is joe's own question and answer